Views/Home/About.cshtml

```diff
@@ -0,0 +1,59 @@
+@{
+    ViewData["Title"] = "About";
+}
+
+
+<!-- Refrence for image postioning  -->
+ <!-- https://stackoverflow.com/questions/67081460/how-can-i-align-image-horizontally-in-tailwind-css -->
```

Comment:
This reference isn't needed, you didn't copy any code from SO they only helped you find the right class names.
Just delete lines 6 and 7. Thanks

Reply:
Ok comments have been removed